lib/web/apiserver.go

```diff
@@ -1540,37 +1546,35 @@ func (h *Handler) find(w http.ResponseWriter, r *http.Request, p httprouter.Para
 		ServerVersion:    teleport.Version,
 		MinClientVersion: teleport.MinClientVersion,
 		ClusterName:      h.auth.clusterName,
+		AutoUpdate:       h.automaticUpdateSettings(r.Context()),
+		Edition:          modules.GetModules().BuildType(),
+		FIPS:             modules.IsBoringBinary(),
 	}
 
-	autoUpdateConfig, err := h.cfg.AccessPoint.GetAutoUpdateConfig(r.Context())
```

Comment:
This (or the new implementation) should 100% not be in the path for `/webapi/find` like this, it's bad enough that we are doing _any_ work in the find handler right now (with `GetProxySettings` and the recent addition of `GetAuthPreference`) - find is rate-unlimited and it's called far too often by basically every cluster component and client.

Maybe this could be the occasion to finally add a `FnCache` with a few seconds of TTL for the object about to be returned? Any per-request personalization can then be added with a shallow clone if necessary.